_config.yml

```diff
@@ -87,6 +87,7 @@ title: "Alembic" # Site name or title, also used in jekyll-seo-tag
 logo: "/assets/logo.svg" # Site logo, also used in jekyll-seo-tag
 description: "Alembic is a starting point for Jekyll projects. Rather than starting from scratch, this boilerplate is designed to get the ball rolling immediately" # Site description and default description, also used in jekyll-seo-tag
 url: "https://alembic.darn.es" # Site url, also used in jekyll-seo-tag
+baseurl: "/baseurl"
```

Comment:
I test with this option turned on and it worked well, however I'd like to have the option empty in the main repo. Can you change it to just `baseurl: ""`, thanks